syntax = "proto3";

message EventLocationMessage {
  int32 userId = 1;
  int32 latitude = 2;
  int32 longitude = 3;
}

service EventLocationService {
  rpc GetServerResponse(EventLocationMessage) returns (EventLocationMessage);
}
